mcda/mcda_run.py

```diff
@@ -245,133 +255,192 @@ def main(input_config: dict, user_input_callback=input):
+            combine_images(images, config.output_file_path,
+                           "MCDA_one_weight_randomness_rough_scores.png")
+            combine_images(images_norm, config.output_file_path,
+                           "MCDA_one_weight_randomness_norm_scores.png")
         logger.info("ProMCDA finished calculations: check the output files")
         elapsed = time.time() - t
         logger.info("All calculations finished in seconds {}".format(elapsed))
     # -------------------------
     # UNCERTAINTY OF INDICATORS
     # -------------------------
-    else: # if is_robustness_indicators == 1
+    else:  # if is_robustness_indicators == 1
         all_indicators_normalized = []
         # cores = config.monte_carlo_sampling["num_cores"] # never used yet
-        if (mc_runs > 0):
-            if (mc_runs < 1000):
-                logger.info("The number of Monte-Carlo runs is only {}".format(mc_runs))
-                logger.info("A meaningful number of Monte-Carlo runs is equal or larger than 1000")
+        if mc_runs > 0:
+            if mc_runs < 1000:
+                logger.info(
+                    "The number of Monte-Carlo runs is only {}".format(mc_runs))
+                logger.info(
+                    "A meaningful number of Monte-Carlo runs is equal or larger than 1000")
                 while True:
-                    user_input = user_input_callback("Do you want to continue (C) or stop (S)? ").strip().lower()
+                    user_input = user_input_callback(
+                        "Do you want to continue (C) or stop (S)? ").strip().lower()
                     if user_input == 'c':
                         break
                     elif user_input == 's':
                         raise UserStoppedInfo()
                     else:
-                        print("Invalid input. Please enter 'C' to continue or 'S' to stop.")
+                        print(
+                            "Invalid input. Please enter 'C' to continue or 'S' to stop.")
             logger.info("Start ProMCDA with uncertainty on the indicators")
-            are_parameters_correct = check_parameters_pdf(input_matrix_no_alternatives, config)
+            are_parameters_correct = check_parameters_pdf(
+                input_matrix_no_alternatives, config)
             if any(not value for value in are_parameters_correct):
-                logger.info('There is a problem with the parameters given in the input matrix with uncertainties. Check your data!')
-                logger.info('Either standard deviation values of normal/lognormal distributed indicators are larger than their means,')
-                logger.info('or max. values of uniform distributed indicators are smaller than their min. values.')
-                logger.info('If you continue, the negative values will be rescaled internally to a positive range.')
+                logger.info(
```

Comment:
Instead of four `logger.info(..)` maybe use multi-line string literal for better readability? Something like:
```
logger.info(''' There is a problem with the parameters given in the input matrix with uncertainties. Check your data!
                      Either standard deviation values of normal/lognormal distributed indicators are larger than their means,
                      or max. values of uniform distributed indicators are smaller than their min. values.
                      ............. ''')